envFiles/.env.ci

```
@@ -106,8 +106,11 @@ REDIS_MAPPED_PORT=6379
-API_OTEL_ENVIRONMENT="local"
-API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
\ No newline at end of file
+API_OTEL_ENABLED=false
+API_OTEL_ENVIRONMENT=ci
+API_OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318/v1/traces
+API_OTEL_SAMPLING_RATIO=1
+API_OTEL_SERVICE_NAME=talawa-api
+
+##### Email Configuration #####
+API_EMAIL_PROVIDER=ses
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)